spec/features/admin_users_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'capybara_helper'
+
+describe Admin::UsersController do
```

Comment:
The test ensuring that is one line below your comment :smile: I am skipping controller specs whenever I can, integration tests cover the whole stack with a little bit more work (and they will get a lot faster with rails 5).